ASTAnalyses/ASTInfo/generator.py

```diff
@@ -0,0 +1,159 @@
+# clang -Xclang -ast-dump -fsyntax-only -fno-color-diagnostics ../test/recover_desugar/main.c > testAST.txt ; python3 ../ASTAnalyses/ASTInfo/generator.py testAST.txt testASTOutput.txt
+
+# python3 ../run_pt.py recover_desugar
+
+import sys
+sys.path.insert(0, '..')
+
+from ASTAnalyses.ASTInfo.AnnoStructure.AnnotationManager import *
+from ASTAnalyses.ASTInfo.Specifiers.FunctionStructure.Function import *
+from ASTAnalyses.ASTInfo.Specifiers.StructStructure.Struct import *
+from ASTAnalyses.ASTInfo.Specifiers.SpecifierManager import *
+from ASTAnalyses.ASTInfo.StructVariables.StructVarManager import *
+from ASTAnalyses.ASTInfo.Debug import *
+
+from ASTAnalyses.ASTInfo.DeclParser.DeclParser import *
+
+file_to_read = sys.argv[1]
+output_file = sys.argv[2]
+
+
+def is_null_stmt(line_of_ast: str):
```

Comment:
Duplicated with `DeclParser.py` around line 350. Please consolidate these so that they share a single implementation.

Reply:
It is unused in `generator.py` so I'll simply delete it from there